property_tests/arbitraries/candid/reference/service_arb/definition_arb.ts

```diff
@@ -80,30 +93,53 @@ function ServiceMethodArb(
             return {
                 name,
                 imports,
-                typeAliasDeclarations,
+                variableAliasDeclarations: variableAliasDeclarations,
```

Comment:
This should probably use object shorthand syntax. The linter should really catch things like this though, see https://eslint.org/docs/latest/rules/object-shorthand.